app/presenters/concerns/hesa_itt_data_api_data.rb

```diff
@@ -35,4 +29,20 @@ def other_ethnicity_details(equality_and_diversity_data)
 
     equality_and_diversity_data['ethnic_background']
   end
+
+  def hesa_codes(equality_and_diversity_data)
+    if application_form.recruitment_cycle_year >= 2023
```

Comment:
would be nice to extract `2023` as a constant, eg `HESA_CODES_UPDATE_YEAR`

Reply:
Done.